openy.install

```diff
@@ -51,14 +51,109 @@ function openy_update_dependencies() {
   $dependencies['openy_node_alert'] = [
     8009 => [
       'system' => 8501,
+      'openy_node_session' => 8010,
```

Comment:
`openy_node_alert` cannot depend on `openy_node_session` due to the fact, `openy_node_session` is not a module from the standard installation type (the module is not required) while `openy_node_alert` is.
Please consider moving it to `openy_node_session.install`